websites/B/BookWalker/dist/metadata.json

```diff
@@ -0,0 +1,30 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "NeroPako",
+		"id": "250864719181643790"
+	},
+	"service": "BookWalker",
+	"description": {
+		"en": "Shows the manga and light novel you're reading on Discord"
```

Comment:
This is not the official description

Reply:
Copied description from https://global.bookwalker.jp/ex/feature/about-us/